src/Shared/SideBar/SideBar.tsx

```diff
@@ -70,6 +70,9 @@ export default function SideBar() {
           </Menu>
         </Sidebar>
 
+
+          </div>
+
         <main style={{ padding: 10 }}>
           <div>
             <div className=" position-relative arrow " onClick={() => setIsCollapsed(!isCollapsed)}>
```

Comment:
We can make a div clickable by adding an onClick event to it, but this is not a good practice. and we are losing a lot of features by not using a button for that like using a tap to navigate between interactive elements, and pressing space to click it, and more.

You should instead make a button and nest the icon inside it, and the button should be giving the onClick event.